### PR TITLE
Added booking date to booking card

### LINKED ISSUE #189: Add booking date to booking card
once this pr https://github.com/kiwicom/graphql/pull/150 is merged

- Add booking date to booking card instead of *Not Available*

screenshot

### PR DESCRIPTION
**Pull request by edgarordonez**
Close #189 

- I added the booking date to booking card.<br/><br/><br/><url>LiveURL: https://kiwicomsmart-faq-yuuqywonia.now.sh
</url>
</url>

### CODE REVIEW

File: src/AllBookings/BookingCard/BookingCard.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ const BookingCard = (props: Props) => {
   const databaseId = idx(props.booking, _ => _.databaseId);
   const passengerCount = idx(props.booking, _ => _.passengerCount);
   const status = props.booking.status && bookingStatus[props.booking.status];
+  const bookingDate = idx(props.booking, _ => _.bookingDate);
```
Comment (git-toni) [inline src/AllBookings/BookingCard/BookingCard.js]: Shouldn't this field `bookingDate`, together with other (unrelated to this PR) fields such as `passengerCount` and `status` be retrieved from `DateAndPassenger`'s fragment? I mean `BookingCard` component doesn't seem to use them.

Comment (jalbertsr) [reply src/AllBookings/BookingCard/BookingCard.js]: yep, that's true, my bad, it can be an issue a part if you want @edgarordonez ?

Comment (edgarordonez) [reply src/AllBookings/BookingCard/BookingCard.js]: Okay, Do I create a new issue, right? @git-toni

Comment (git-toni) [reply src/AllBookings/BookingCard/BookingCard.js]: That would be ideal I guess :slightly_smiling_face:.

I'm also thinking we should evaluate the `null`ability of those fields, and maybe provide `idx(...) | something|`? Except `bookingDate`, which I believe cannot be `null` nor empty being of `GraphQLDate` type?

Comment (edgarordonez) [reply src/AllBookings/BookingCard/BookingCard.js]: Okay, so I'm gonna to create a new issue to solve it. 😃

Comment (jaroslav-kubicek) [reply src/AllBookings/BookingCard/BookingCard.js]: >  Except bookingDate, which I believe cannot be null

can be null as well, nearly everything can be null in GraphQL 💥

File: src/AllBookings/BookingCard/__test__/DateAndPassenger.test.js [hunk 2]
```diff
@@ -16,8 +16,15 @@ const departure = {
 
 const status = bookingStatus['CONFIRMED'];
 
+const bookingDate = '2017-11-23T20:40:00.000Z';
```
Comment (jaroslav-kubicek) [inline src/AllBookings/BookingCard/__test__/DateAndPassenger.test.js]: I would suggest removing these two variables and pass values directly to the component - or at least remove the empty line on line 18, my OCD meter is broken :D

**Comment by jalbertsr** [comment]
with this https://github.com/kiwicom/smart-faq/pull/231 merged we can close this PR

**Pull request closed without merging**